index.html

```diff
@@ -2,472 +2,488 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Sukoon</title>
-  <link rel="icon" href="/images/favicon-enhanced.png">
-  <link rel="stylesheet" href="css/style.css" />
-  <link rel="stylesheet" text="text/CSS"
-    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sukoon</title>
+    <link rel="icon" href="/images/favicon-enhanced.png">
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/assets/owl.carousel.min.css">
+
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" text="text/CSS" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
 </head>
 <style>
-  html {
-    scroll-behavior: smooth;
-  }
-
-  .hidden {
-    visibility: hidden;
-  }
-
-  .back-top-wrap {
-    position: fixed;
-    bottom: 5rem;
-    right: 1rem;
-    background-color: #80F7D1;
-    padding: 1rem 1.2rem;
-    border-radius: 50%;
-    cursor: pointer;
-    transition: .3s ease-in-out;
-    z-index: 500;
-  }
-
-  .back-top-wrap:hover {
-    background-color: #00a2ff;
-    color: #80F7D1;
-  }
-
-  .back-top-wrap:hover .fas.fa-arrow-up {
-    color: white;
-  }
-
-  .fas.fa-arrow-up {
-    color: #00488F;
-    transition: 0.3s ease-in-out;
-  }
+    html {
+        scroll-behavior: smooth;
+    }
+    
+    .hidden {
+        visibility: hidden;
+    }
+    
+    .back-top-wrap {
+        position: fixed;
+        bottom: 5rem;
+        right: 1rem;
+        background-color: #80F7D1;
+        padding: 1rem 1.2rem;
+        border-radius: 50%;
+        cursor: pointer;
+        transition: .3s ease-in-out;
+        z-index: 500;
+    }
+    
+    .back-top-wrap:hover {
+        background-color: #00a2ff;
+        color: #80F7D1;
+    }
+    
+    .back-top-wrap:hover .fas.fa-arrow-up {
+        color: white;
+    }
+    
+    .fas.fa-arrow-up {
+        color: #00488F;
+        transition: 0.3s ease-in-out;
+    }
 </style>

```

Comment:
Instead of writing these CSS codes that are very important, you can make a separate **CSS file** and name it `global.css`
It'd be great.

Reply:
@AliReza1083 Do you mean a single `global.css` file having the style of all the pages?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Yeah, Those codes which are very very Important, and will not be changed later, You can write in `global.css`.
Your codes will look clean and clear.